### DISCUSSION TITLE
Airflow Connections page not opening

### DISCUSSION THREAD

**Discussion by bcgvbapathi** [apache/airflow#34345] [Q&A]
Hi,

My Airflow instance running on Docker is working fine for all other functionalities but the connections page is not opening and not able to add new connections or see existing connections.
![Screenshot 2023-09-12 at 5 17 13 PM](https://github.com/apache/airflow/assets/142817655/b4210ee1-4100-469e-bf69-03a65a001a2b)

**Comment by potiuk** [comment]
Did you follow the advice  and checked for logs in the wesbserver ? Without you following it, it is impossible to help you with your problem.

**Comment by potiuk** [comment]
Hint: "gather the relevant information"  is actually telling you what you should do.

**Comment by bcgvbapathi** [comment]
Issue has been resolved after doing docker prune and restarting docker
